platform/server/src/main/webapp/WEB-INF/jsp/bootstrapconfig.jsp

```diff
@@ -23,7 +21,7 @@
         </script>
     </c:if>
 </head>
-<body class="body-startup" ng-app="bootstrapApp" >
+<body id="mainBody" class="body-startup" ng-app="bootstrapApp" ng-controller="controller" >
```

Comment:
Controller name is too general. Maybe better will be 'formController' or 'bootstrapFormController'.